server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java

```diff
@@ -216,15 +217,29 @@ public void deleteUser(String userId) throws DBCException {
     public void setUserTeams(String userId, String[] teamIds, String grantorId) throws DBCException {
         try (Connection dbCon = database.openConnection()) {
             try (JDBCTransaction txn = new JDBCTransaction(dbCon)) {
-                setUserTeams(dbCon, userId, teamIds, grantorId);
-                txn.commit();
+                if (containsAllUsers(teamIds)) {
```

Comment:
just always add default team to user's team list, and we shouldn't throw an error, otherwise it will break SSO login